File: backend/core/src/main/java/com/feed_grabber/core/rabbit/entityExample/CloseRequest.java
package com.feed_grabber.core.rabbit.entityExample;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.Date;
import java.util.UUID;

@AllArgsConstructor
@Data
public class CloseRequest {
    private UUID requestId;
    private Date closeDate;
}

File: backend/event_processor/src/main/kotlin/com/feed_grabber/event_processor/rabbit/entityExample/CloseRequest.kt
package com.feed_grabber.event_processor.rabbit.entityExample

import java.util.*

class CloseRequest {
    private val requestId: UUID = UUID.randomUUID()
    private val closeDate: Date = Date()

    fun getRequestId(): UUID = requestId

    fun getCloseDate(): Date = closeDate
}